test/user.test.js

```diff
@@ -363,6 +363,88 @@ describe('User', function() {
     });
   });
 
+  describe('Password length validation', function() {
+    var pass72Char = new Array(73).join('a');
+    var pass73Char = new Array(74).join('a');
+    var passTooLong = pass72Char + 'WXYZ1234';
```

Comment:
basically we need the first 72 chars to be identical with passTooLong, and `Array(80)` would still fulfill that because the word is all `a` but I wanted something different starting at the 73rd index so we can show how the test fails.